packages/@divvi/mobile/src/home/DivviBottomSheet.tsx

```diff
@@ -0,0 +1,197 @@
+import Button, { BtnSizes, BtnTypes } from 'src/components/Button'
+import { divviBottomSheetSeen } from 'src/home/actions'
+import { showDivviBottomSheetSelector } from 'src/home/selectors'
+import { divviPie } from 'src/images/Images'
+import Logo from 'src/images/Logo'
+import LogoHeart from 'src/images/LogoHeart'
+import { navigate } from 'src/navigator/NavigationService'
+import { Screens } from 'src/navigator/Screens'
+import { useDispatch, useSelector } from 'src/redux/hooks'
+import Colors from 'src/styles/colors'
+import { typeScale } from 'src/styles/fonts'
+import { Spacing } from 'src/styles/styles'
+
+const DIVVI_SLICES_URL = 'https://slices.divvi.xyz'
+
+export default function DivviBottomSheet() {
+  const dispatch = useDispatch()
+  const { t } = useTranslation()
+
+  const insets = useSafeAreaInsets()
+  const insetsStyle = { paddingBottom: Math.max(insets.bottom, Spacing.Regular16) }
+
+  const bottomSheetRef = useRef<BottomSheetModalRefType>(null)
+
+  const shouldShowBottomSheet = useSelector(showDivviBottomSheetSelector)
+
+  const handleBottomSheetPositionChange = (index: number) => {
+    if (index === -1) {
+      AppAnalytics.track(HomeEvents.divvi_bottom_sheet_displayed)
+      dispatch(divviBottomSheetSeen())
+    }
+  }
+
+  const handleCtaPress = () => {
+    AppAnalytics.track(HomeEvents.divvi_bottom_sheet_cta_pressed)
+    bottomSheetRef.current?.close()
+    navigate(Screens.WebViewScreen, { uri: DIVVI_SLICES_URL })
+  }
+
+  const handleBottomSheetClose = () => {
+    dispatch(divviBottomSheetSeen())
+  }
+
+  useEffect(() => {
+    if (shouldShowBottomSheet) {
+      const timeoutId = setTimeout(() => {
+        bottomSheetRef.current?.expand()
+      }, 1000)
+      return () => clearTimeout(timeoutId)
+    }
+  }, [shouldShowBottomSheet, bottomSheetRef.current])
+
+  if (!shouldShowBottomSheet) {
+    return null
+  }
+
+  return (
+    <BottomSheetBase
+      forwardedRef={bottomSheetRef}
+      handleComponent={() => null} // handle is rendered within the content body
+      backgroundStyle={styles.bottomSheetBackground}
+      onChange={handleBottomSheetPositionChange}
+      onClose={handleBottomSheetClose}
+    >
+      <BottomSheetView style={[insetsStyle, styles.sheetContainer]}>
+        <View style={styles.topSection}>
+          <View style={styles.handleBar} />
+          <View style={styles.iconRow}>
+            <LogoHeart size={72} />
+            <Text style={styles.iconPlus}>+</Text>
+            <Logo
+              size={56}
+              color={Colors.secondaryAccent}
+              backgroundColor={Colors.contentPrimary}
+            />
+            <Text style={styles.iconEquals}>=</Text>
+            <Text style={styles.emoji}>🎉</Text>
+          </View>
+        </View>
+        <View style={styles.bottomSection}>
+          <Text style={styles.title}>{t('divviBottomSheet.title')}</Text>
+          <View style={styles.descriptionRow}>
+            <Text style={styles.description}>
```

Comment:
This is pretty ugly -- I tried using a `Trans` component and parameterizing the translation string with the SVG components, but apparently there are limitations in RN for rendering SVGs within `Text` components, and I was unable to get their alignment correct within the string. If anyone knows a better way of doing this, please lmk!